samples/samples/admin/pg_samples.py

```diff
@@ -0,0 +1,351 @@
+#!/usr/bin/env python
+
+# Copyright 2024 Google, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This application demonstrates how to do basic operations using Cloud
+Spanner PostgreSql dialect.
+For more information, see the README.rst under /spanner.
+"""
+from google.cloud import spanner
+from google.cloud.spanner_admin_database_v1.types.common import DatabaseDialect
+
+OPERATION_TIMEOUT_SECONDS = 240
+
+
+# [START spanner_postgresql_create_database]
+def create_database(instance_id, database_id):
+    """Creates a PostgreSql database and tables for sample data."""
+
+    from google.cloud.spanner_admin_database_v1.types import spanner_database_admin
+
+    spanner_client = spanner.Client()
+    instance = spanner_client.instance(instance_id)
+
+    request = spanner_database_admin.CreateDatabaseRequest(
+        parent=instance.name,
+        create_statement=f'CREATE DATABASE "{database_id}"',
+        database_dialect=DatabaseDialect.POSTGRESQL,
+    )
+
+    operation = spanner_client.database_admin_api.create_database(request=request)
```

Comment:
@rahul2393 
Do we need to take care of client closure as discussed in Nodejs?
https://github.com/googleapis/nodejs-spanner/pull/1994#discussion_r1489539854

Reply:
We are only initializing admin clients once per client object lifecycle
https://github.com/googleapis/python-spanner/blob/main/google/cloud/spanner_v1/client.py#L244

There is __exit__ method in all of the autogenerated classes which get's called when object is ready for garbage collection.